library/pk.c

```diff
@@ -906,6 +906,8 @@ int mbedtls_pk_wrap_as_opaque(mbedtls_pk_context *pk,
             return PSA_PK_TO_MBEDTLS_ERR(status);
         }
 
+        mbedtls_platform_zeroize(d, sizeof(d));
```

Comment:
Thanks for noticing this was missing and adding it! However I think this should also be done when `psa_import_key()` fails before returning. Perhaps just move it between that call and the corresponding `if (status` check?